File: DAA/Lab/GCD.go
package main

import "fmt"

func main() {
	var a, b, temp int
	fmt.Println("enter 2 nums")
	fmt.Scanln(&a)
	fmt.Scanln(&b)
	for {
		if b == 0 {
			break
		}
		temp = a % b
		a = b
		b = temp
	}
	fmt.Println("GCD from Go: ", a)
}

File: DAA/Lab/GCD.cpp
#include <iostream>
#include <ostream>
int main(void) {

  int a, b, temp;
  std::cout << "Enter 2 num\n";
  std::cin >> a;
  std::cin >> b;

  while (b != 0) {
    temp = a % b;
    a = b;
    b = temp;
  }
  std::cout << "GCD: " << a << "\n";
  return 0;
}
